tests/PyroTests/test_httpgateway.py

```diff
@@ -18,6 +18,9 @@
 # a bit of hackery to avoid having to launch a live name server
 def get_nameserver_dummy(hmac=None):
     class NameServerDummyProxy(NameServer):
+        class Proxy(object):
+            object = "PYRO:dummy12345@localhost:59999"
+        _pyroUri = Proxy
```

Comment:
this doesn't seem right, shouldn't it be a PyroURI object instead

Reply:
I've updated to use the real class rather than mocking the missing attribute